recipe/meta.yaml

```diff
@@ -53,13 +53,14 @@ requirements:
     - pkg-config  # [not win]
   host:
     - python
-    - sip >=4.19.4,<=4.19.8
+    - enum34  # [py27]
+    - sip 4.19.18
     - dbus  # [not win]
-    - qt 5.9.*
+    - qt 5.12
```

Comment:
it should be `5.12.*`, else conda emits a warning

Reply:
Ok, I thought it was the other way around. Will fix it.